app/src/main/res/layout/item_album.xml

```diff
@@ -62,33 +77,40 @@
                 android:layout_marginTop="15dp"
                 android:textAppearance="?android:attr/textAppearanceLarge"
                 android:textColor="@color/piwigo_orange"
+                tools:text="peterdqdwqdqwdqdwqdqw"
```

Comment:
This placeholder is kinda weird.. Maybe go for something more simple like "textview" ?

Reply:
Or even better would be to put the 'content' here... "Album title"

Reply:
Wow this looks nice!

> Also, we can only open an album by clicking on the picture, when we should be able to open it by clicking anywhere on the ticket.
> 
> Finally, I don't know @ramack's opinion about this, but you may want to close this PR and open another one with only the needed commits :)

Yes, I think a fray PR with just the interesting commits would be best, but for sure we can squash the PR and then it'll become a single commit during 'merge'...